client/js/common.js

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable no-unused-vars */
```

Comment:
IDK how to fix this properly either, but can you use:
```
// eslint-disable-next-line no-unused-vars
```
instead? Easy one liner 😄